optuna/integration/allennlp.py

```diff
@@ -373,6 +373,18 @@ def run(self) -> float:
         for package_name in self._include_package:
             allennlp.common.util.import_module_and_submodules(package_name)
 
+        # Without the following lines, the transformer model construction only takes place in the
+        # first trial (which would consume some random numbers), and the cached model will be used
+        # in trials afterwards (which would not consume random numbers), leading to inconsistent
+        # results between single trial and multiple trials. To make results reproducible in
+        # multiple trials, we clear the cache before each trial.
+        # TODO(MagiaSN) When AllenNLP has introduced a better API to do this, one should remove
+        # these lines and use the new API instead. For example, use the `_clear_caches()` method
+        # which will be in the next AllenNLP release after 2.4.0.
+        import allennlp.common.cached_transformers
```

Comment:
Please move this import under `try_imports`.
https://github.com/optuna/optuna/blob/29045176b0b16f25a141e8c616be5d78c00cf199/optuna/integration/allennlp.py#L22

Reply:
Ok, donw